src/modules/aws_sdk/migration/aws_sdk.ts

```diff
@@ -0,0 +1,10 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
do we need this?

Reply:
Yes it does. If it doesn't exist the system will break. So I created this dummy migration.